contracts/staking/contracts/src/immutable/MixinStorage.sol

```diff
@@ -151,6 +146,9 @@ contract MixinStorage is
     /// @dev State for unfinalized rewards.
     IStructs.UnfinalizedState public unfinalizedState;
 
+    /// @dev The WETH balance of this contract that is reserved for pool reward payouts.
+    uint256 _reservedWethBalance;
```

Comment:
Nit - we may be able to communicate what this is reserved for in the name, like `_wethReservedForPools` or something.